R/expr__expr.R

```diff
@@ -1489,9 +1567,9 @@ Expr_gather = function(indices) {
 #' @examples
 #' pl$DataFrame(a = c(1, 2, 4, 5, 8))$
 #'   with_columns(
-#'     pl$col("a")$shift(-2)$alias("shift-2"),
-#'     pl$col("a")$shift(2)$alias("shift+2")
-#'   )
+#'   pl$col("a")$shift(-2)$alias("shift-2"),
+#'   pl$col("a")$shift(2)$alias("shift+2")
+#' )
```

Comment:
isn't this indentation wrong? We do want some additional indentation when we are in `with_columns()` (I guess this is `styler`'s fault)

Same thing for all other changes like this

Reply:
Certainly.
We'll probably need to create an issue in the styler repository.